lua/hbac/init.lua

```diff
@@ -92,7 +92,38 @@ local function close_unused()
 	end
 end
 
+
+-- Create commands to add and remove buffers from the persistant_buffers table
+M.release_buffer = function(bufnr)
+	bufnr = bufnr or vim.api.nvim_get_current_buf()
+	M.persistant_buffers[bufnr] = nil
+end
+
+M.commands = {
+	pin = M.persist_buffer,
+	unpin = M.release_buffer,
+	close_unused = close_unused,
+}
+
+function M.hbac_cmd(arg)
+	if M.commands[arg] then
+		M.commands[arg]()
+		print("Hbac: Buffer " .. arg .. "ned")
+	else
+		print("Unknown Hbac command: ", arg)
+	end
+end
+
+vim.api.nvim_create_user_command("Hbac",
+	function(args)
+		M.hbac_cmd(args.args)
+	end,
+	{
+	nargs = 1,
```

Comment:
```suggestion
	nargs = 1,
	complete = function()
		-- return completion candidates as a list-like table
		return {[1] = "pin", [2] = "unpin", [3] = "close_unused"}
	end,
```

Your suggestion works perfectly but my LSP was complaining without the brackets. (`lua_ls` 3.6.18). Yours looks better but just thought I'd mention it. Is it for Lua 5.1 compatibility?

Reply:
Lol, no. I'm just not that good at lua. Also is it possible to just write `return {"pin", "unpin", "close_unused"}` without the indexes?